tests/utils.py

```diff
@@ -180,3 +180,54 @@ def set_dict_element(data, path, value, delimiter='.'):
     path, _, key = path.rpartition(delimiter)
     (get_dict_element(data, path) if path else data)[key] = value
     return data
+
+
+def get_grain(host, key):
+    with host.sudo():
+        output = host.check_output(
+            'salt-call --local --out=json grains.get "{}"'.format(key)
+        )
+        grain = json.loads(output)['local']
+
+    return grain
+
+
+class PrometheusApiError(Exception):
+    pass
+
+
+class PrometheusApi():
+    def __init__(self, host, port=CONTROL_PLANE_INGRESS_PORT):
```

Comment:
If we really want a default value, maybe use the default Prometheus port (9090) and not the Ingress?